app/controllers/employers/employer_attestations_controller.rb

```diff
@@ -25,6 +25,14 @@ def verify_attestation
     @document = attestation.employer_attestation_documents.find(params[:employer_attestation_id])
   end
 
+  def revert_attestation
+    attestation = @employer_profile.employer_attestation
+    @document = attestation.employer_attestation_documents.find(params[:employer_attestation_id])
+    @document.submit! if @document.may_submit?
+    attestation.submit! if attestation.may_submit?
+    redirect_to employer_invoice_exchanges_hbx_profiles_path
+  end
+
```

Comment:
could you add a controller spec for this?

set up the employer_profile, attestation, and document. 

Use a before block to allow employer_attestation_documents accept the id param and return a mock document. 

Test the case where may_submit? is false and true. for each context you're expecting document.to receive(:submit!) or not_to receive(:submit!)

either way you should expect the redirect.